src/browser/modules/Stream/Stream.tsx

```diff
@@ -21,71 +21,15 @@
 import { connect } from 'react-redux'
```

Comment:
Nice to see the restructuring eliminates functions that shouldn't be this component's responsibility.